### ISSUE TITLE
Unable to mouseover on scatter plot and display the title rendering

### ISSUE THREAD

**Issue by louishenri13** [dc-js/dc.js#961]
The title does not render when I use scatter plot without brush. It seems to be working with all the other graphs but scatter plot. This person has had the same question for almost 2 years and it does not seem to be resolved: "Tooltip not rendered on Scatterplots #463" (amergin, 2013). I was wandering if someone could potentially fix this issue.

Thank you,
Louis-Henri

**Comment by ericgross** [comment]
:+1:

**Comment by maxwang7** [comment]
Any updates on this issue? If not, I'll work on a PR to fix it.

**Comment by gordonwoodhull** [comment]
Yes, please contribute a fix! This is a community-driven project.

**Comment by maxwang7** [comment]
Cool. Thoughts on why this could be happening?

**Comment by gordonwoodhull** [comment]
Not much of the actual rendering code is shared between charts. Looks like the title was never implemented here - take a look at how it's appended to the corresponding elements in the other charts.

**Comment by daniel-gall** [comment]
Submitted pull request for this.

https://github.com/dc-js/dc.js/pull/1198

**Comment by daniel-gall** [comment]
Closed original pull request and opened:

https://github.com/dc-js/dc.js/pull/1200
